src/cloud-cli/applications/configure.ts

```diff
@@ -42,9 +43,18 @@ export async function configureApp(host: string, port: string, dbName: string) {
       return;
     }
 
+    try {
+      // Should we just download and keep the file in the dir instead of downloading everytime ??
```

Comment:
We can do an "if file not exists" check here.

Reply:
No issue if the file exists. We just overwrite it. No big deal. My question was -- should keep the file in the examples/hello so we never make the rest call ? The downside is that if they change the cert chain , then the file we have might be out of date.

Reply:
Oh, I see, I don't think we should keep the file there, downloading it is fine as you said